packages/api/src/apps/api/README.md

```diff
@@ -96,3 +96,11 @@ Get name, decimals of an erc20 by address.
 ### getAllErc20Info() => Erc20Data[]
 
 List all known erc20s, collateral or synthetic and all known information about them
+
+### getEmpStats(address: string, currency: "usd" = "usd") => StatData
```

Comment:
Do you think we'll do something similar with LSPs?

Reply:
what is a LSP?

Reply:
oh the CFD contracts, yea totally we could add stats about it